packages/blade/src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/prefer-ts-expect-error */
+import React from 'react';
+import { ThumbIcon } from './ThumbIcon';
+import type { SwitchProps } from './types';
+import { switchHoverTokens } from './switchTokens';
+import { Thumb } from './Thumb';
+import { AnimatedThumb } from './AnimatedThumb';
+import { SwitchTrack } from './SwitchTrack';
+import { useTheme } from '~components/BladeProvider';
+import BaseBox from '~components/Box/BaseBox';
+import { useCheckbox } from '~components/Checkbox/useCheckbox';
+import { SelectorInput } from '~components/Form/Selector/SelectorInput';
+import {
+  assignWithoutSideEffects,
+  makeAccessible,
+  metaAttribute,
+  useBreakpoint,
+  usePrevious,
+} from '~utils';
+import type { BladeElementRef } from '~src/hooks/types';
+import { SelectorLabel } from '~components/Form/Selector/SelectorLabel';
+
+const _Switch: React.ForwardRefRenderFunction<BladeElementRef, SwitchProps> = (
+  {
+    defaultChecked,
+    isChecked,
+    isDisabled,
+    name,
+    onChange,
+    size = 'medium',
+    value,
+    accessibilityLabel,
+    id,
```

Comment:
We decided to remove `id` in API decision

Reply:
done.